backend/samfundet/models/recruitment.py

```diff
@@ -180,13 +180,53 @@ class RecruitmentAdmission(CustomBaseModel):
         choices=RecruitmentStatusChoices.choices, default=RecruitmentStatusChoices.NOT_SET, help_text='The status of the admission'
     )
 
+    def organize_priorities(self) -> None:
+        """Organizes priorites from 1 to n, so that it is sequential with no gaps"""
+        admissions_for_user = RecruitmentAdmission.objects.filter(recruitment=self.recruitment, user=self.user).order_by('applicant_priority')
+        for i in range(len(admissions_for_user)):
+            if admissions_for_user[i].applicant_priority != i + 1:
+                admissions_for_user[i].applicant_priority = i + 1
+                admissions_for_user[i].save()
+
+    def update_priority(self, direction: int) -> None:
+        """
+        Method for moving priorites up or down,
+        positive direction indicates moving it to higher priority,
+        negative direction indicates moving it to lower priority,
+        can move n positions up or down
+
+        """
+        admissions_for_user = RecruitmentAdmission.objects.filter(recruitment=self.recruitment, user=self.user)
+        # Use order for more simple an unified for direction
+        admissions_for_user = admissions_for_user.order_by('-applicant_priority') if (direction > 0) else admissions_for_user.order_by('applicant_priority')
```

Comment:
Tror det er lettere å isolere det som faktisk er differansen
```suggestion
        ordering = f"{'-' if direction < 0 else '' }applicant_priority"
        admissions_for_user = admissions_for_user.order_by(ordering)
```